### PR TITLE
The correct display of the sun icon in the Streetcode catalog has been fixed

### PR DESCRIPTION
**Pull request by ValDekh**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## #909 

ToDo

## Summary of change

Change StreetcodeCatalog.component.tsx

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.component.tsx [hunk 1]
```diff
@@ -27,6 +28,11 @@ const StreetcodeCatalog = () => {
 
     useAsync(async () => {
         const count = await StreetcodesApi.getCount();
+        const fetchedPublishCount = (await StreetcodesApi.getAllPublished()).length;
+        setPublishCount(fetchedPublishCount);
+        if (publishCount === 8) {
```
Comment (Adeodonne) [inline src/features/StreetcodeCatalogPage/StreetcodeCatalog.component.tsx:33]: maybe, create constant in another file and import in this component

Comment (ValDekh) [reply src/features/StreetcodeCatalogPage/StreetcodeCatalog.component.tsx:33]: In my opinion, this case will complicate code clarity. Here this constant clearly shows all fetched Streetcodes.

Comment (malyna2) [reply src/features/StreetcodeCatalogPage/StreetcodeCatalog.component.tsx:33]: At least create a separate const variable

**Review by malyna2 — APPROVED** [review]

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.component.tsx (hunk 1 above)
Comment (ValDekh) [reply src/features/StreetcodeCatalogPage/StreetcodeCatalog.component.tsx:33]: publishCount is separate const variable which changed by useState hook

**Pull request merged**